Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class ConsoleLogger : ILogger
+    {
+        public void Error(Exception ex)
+        {
+            Console.WriteLine($"{System.DateTime.Now} - Exception has been thrown: {ex} {Environment.NewLine}");
+        }
+
+        public void Error(Exception ex, string message)
+        {
+            Error(ex);
+            Console.WriteLine(message);
+        }
+
+        public void LogExit()
+        {
+            Console.WriteLine($"{System.DateTime.Now} - Program exit");
+        }
+
+        public void LogStart()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'LogStart' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5141367)